src/compiler/parser.rs

```diff
@@ -194,6 +234,37 @@ impl ParserType {
         node
     }
 
+    fn create_missing_node(
```

Comment:
The `createMissingNode()` helper returns multiple different AST node subtypes depending on what it's passed so I represented that with a wrapping `MissingNode` enum that I then unwrap below to extract the inner `Identifier`